### PR TITLE
7 migration from react native to nextjs codebase

### PR DESCRIPTION
**Pull request by Stuartwastaken**
I pulled in the code like user class, media class types, async storage, types.ts (has note a media type). I also branched from izaks setup with tailwind and nextjs13.

### CODE REVIEW

File: .DS_Store
Comment (kate-holdener) [inline .DS_Store:1]: Please make sure .DS_Store files don't get merged in

File: .next/build-manifest 2.json
Comment (kate-holdener) [inline .next/build-manifest 2.json:1]: These are redundant files that seem to have gotten here accidentally

File: .next/cache/webpack/client-development-fallback/.0.pack.gz.icloud
Comment (kate-holdener) [inline .next/cache/webpack/client-development-fallback/.0.pack.gz.icloud:1]: No binary files in the repo, please

**Comment by izakrobles** [comment]
good starting point, we just need to get the logic to work

**Comment by yashb196** [comment]
please fix the conflict from package.json

**Review by izakrobles — COMMENTED** [review]
looks good m8 10/10 gitignore file

**Review by yashb196 — APPROVED** [review]
Fixed the merge conflicts and approving the request

**Pull request merged**